### PR TITLE
[DOCS] added some arvr docs

### PR DESCRIPTION
**Pull request by Grosskopf**
that was some well documented stuff :) easy job :)

### CODE REVIEW

File: doc/classes/ARVRController.xml [hunk 1]
```diff
@@ -31,6 +31,7 @@
 			<return type="int" enum="ARVRPositionalTracker.TrackerHand">
 			</return>
 			<description>
+				Returns if known, the hand this controler is held in. See TRACKER_* constants in [ARVRPositionalTracker].
```
Comment (Noshyaar) [inline doc/classes/ARVRController.xml]: controller

File: doc/classes/ARVRAnchor.xml [hunk 2]
```diff
@@ -60,6 +60,7 @@
 	</methods>
 	<members>
 		<member name="anchor_id" type="int" setter="set_anchor_id" getter="get_anchor_id">
+			The anchor's id. You can set this before the anchor itself exists. The first anchor that gets an id of [code]1[/code], the second an id of [code]2[/code], etc. When anchors get removed the corresponding id stays free. The engine can then assign it to new anchors. The most common situation where anchors 'disappear' is when the AR server identifies that two anchors represent different parts of the same plane and merge them.
```
Comment (cbscribe) [inline doc/classes/ARVRAnchor.xml]: - "The first anchor gets an id"
- merge - merges
- These 2 sentences could be combined:
> When anchors get removed the corresponding id stays free. The engine can then assign it to new anchors. 

When anchors are removed the corresponding id remains free and the engine can assign it to new anchors.

File: doc/classes/ARVRController.xml [hunk 3]
```diff
@@ -31,6 +31,7 @@
 			<return type="int" enum="ARVRPositionalTracker.TrackerHand">
 			</return>
 			<description>
+				Returns if known, the hand this controller is held in. See TRACKER_* constants in [ARVRPositionalTracker].
```
Comment (cbscribe) [inline doc/classes/ARVRController.xml]: Returns the hand holding this controller, if known.

File: doc/classes/ARVRController.xml [hunk 4]
```diff
@@ -77,6 +78,7 @@
 	</methods>
 	<members>
 		<member name="controller_id" type="int" setter="set_controller_id" getter="get_controller_id">
+			The controller's id. The first controller that the ARVR Server detects will have id 1, the second id 2, the third id 3, etc. When a controller is turned off that slot is freed ensuring that controllers will keep the same id while it is turned on even when controllers with lower ids are turned off.
```
Comment (cbscribe) [inline doc/classes/ARVRController.xml]: - Some tense problems here:
> When a controller is turned off that slot is freed ensuring that controllers will keep the same id while it is turned on even when controllers with lower ids are turned off.

When a controller is turned off, its slot is freed. This ensures controllers will keep the same id even when controllers with lower ids are turned off.

File: doc/classes/ARVROrigin.xml [hunk 5]
```diff
@@ -35,6 +35,8 @@
 	</methods>
 	<members>
 		<member name="world_scale" type="float" setter="set_world_scale" getter="get_world_scale">
+			This scale allows you to adjust this to the unit system you use in your game. Most AR/VR platforms will assume a unit size of 1 unit in your game world = 1 meter in the real world.
```
Comment (cbscribe) [inline doc/classes/ARVROrigin.xml]: - Allows you to adjust the scale to your game's units.
- platforms will assume -> platforms assume
-  a unit size of 1 unit in your game world = 1 meter in the real world

maybe clearer as
...assume a scale of 1 game world unit = 1 meter in the real world.

File: doc/classes/ARVRPositionalTracker.xml [hunk 6]
```diff
@@ -17,6 +17,7 @@
 			<return type="int" enum="ARVRPositionalTracker.TrackerHand">
 			</return>
 			<description>
+				Returns if known, the hand this tracker is held in. See TRACKER_* constants.
```
Comment (cbscribe) [inline doc/classes/ARVRPositionalTracker.xml]: Returns the hand holding this tracker, if known.

File: doc/classes/ARVRPositionalTracker.xml [hunk 7]
```diff
@@ -80,10 +81,13 @@
 	</methods>
 	<constants>
 		<constant name="TRACKER_HAND_UNKNOWN" value="0">
+			The hand this tracker is held in is unknown or not applicable
```
Comment (cbscribe) [inline doc/classes/ARVRPositionalTracker.xml]: Missing periods on this and all constants below.

File: doc/classes/ARVRServer.xml [hunk 8]
```diff
@@ -121,6 +121,7 @@
 	</methods>
 	<members>
 		<member name="world_scale" type="float" setter="set_world_scale" getter="get_world_scale">
+			The world scale, see the ARVROrigin documentation for more information.
```
Comment (cbscribe) [inline doc/classes/ARVRServer.xml]: Don't just repeat the member name as the description. See above scale description.

Is there a link to the ARVROrigin documentation?

**Review by cbscribe — CHANGES_REQUESTED** [review]

File: doc/classes/ARVRServer.xml (hunk 8 above)
Comment (Grosskopf) [reply doc/classes/ARVRServer.xml]: i should really read more of what i commit, sorry :/

File: doc/classes/ARVRController.xml [hunk 9]
```diff
@@ -77,6 +78,7 @@
 	</methods>
 	<members>
 		<member name="controller_id" type="int" setter="set_controller_id" getter="get_controller_id">
+			The controller's id. The first controller that the ARVR Server detects will have id 1, the second id 2, the third id 3, etc. When a controller is turned off, it's slot is freed. This ensures controllers will keep the same id even when controllers with lower ids are turned off.
```
Comment (cbscribe) [inline doc/classes/ARVRController.xml]: [ARVRServer]

File: doc/classes/ARVROrigin.xml [hunk 10]
```diff
@@ -35,6 +35,8 @@
 	</methods>
 	<members>
 		<member name="world_scale" type="float" setter="set_world_scale" getter="get_world_scale">
+			Allows you to adjust the scale to your game's units. Most AR/VR platforms assume a scale of 1 game world unit = 1 meter in the real world.
+			Note that this method is a passthrough to the ARVRServer itself.
```
Comment (cbscribe) [inline doc/classes/ARVROrigin.xml]: [ARVRServer]

**Comment by cbscribe** [comment]
Great work, thanks!

**Review by cbscribe — APPROVED** [review]

**Pull request merged**